### PR TITLE
NoSerializationVerification for local discovery messages

### PR DESCRIPTION
**Pull request by chbatey**
Refs #26788

### CODE REVIEW

**Comment by patriknw** [comment]
In some other modules we have `akka.actor.serialize-messages=on` in `src/test/resources/reference.conf`

**Comment by chbatey** [comment]
That makes sense, i'll add this to this PR

File: akka-discovery/src/test/resources/reference.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+akka.actor.serialize-messages=on
```
Comment (patriknw) [inline akka-discovery/src/test/resources/reference.conf]: in those it is important to have the trailing line breaks (probably doesn't matter here, but better to not risk trouble)

**Review by patriknw — APPROVED** [review]
LGTM, after linebreak

**Review by helena — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
Remember to backport!

**Pull request merged**